test/py/test_module_construct.py

```diff
@@ -0,0 +1,64 @@
+import migraphx
+import numpy as np
```

Comment:
I think it should be fine to just check for numpy that it's installed, and exit if it's not. Our CI runs on Jenkins will test with numpy, but Github actions will not.